script/setup/install-seccomp

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+#   Copyright The containerd Authors.
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+
+#
+# Builds and installs runc to /usr/local/go/bin based off
+# the commit defined in vendor.conf
```

Comment:
This doesn't seem like the right comment.

Reply:
That's a mistake in master from the original commit. I just opened a PR to fix.

Reply:
Ah on my phone didn't notice this was a backport.

Reply:
Ah on my phone didn't notice this was a backport.

Reply:
Fixed the comment also. Thanks :)